### DISCUSSION TITLE
Luban Linux install

### DISCUSSION THREAD

**Discussion by ipeco85** [Snapmaker/Luban#1663] [Q&A]
Hi, 
I'm Peter living in Vienna/Austria and try to make some first steps in using my Snapmaker A250 using Linux Mint. The quite old but well working SONY Vaio notebook had some issues using the WIN software. So I'd like to try if it would work better using Linux.
Working for about 30 years using DOS and WINx software I'm unfortunately completely unexperience with Linux. My question is how to get the linux 4.3.2 x64 tar.gz packet working on my notebook.
Download is done and a copy of the file lives on the Mint desktop. What's going to do next? Is it necessary to install usb drivers for the snapmaker or are they included in the packages? Question over questions ..
Probably there already exist a Linux dummies guide for installing Luban.
Many thanks for your help
Peter

**Comment by ipeco85** [comment]
Problem solved

**Reply by sniper614** [reply]
what was the FIX?

**Reply by yragnos** [reply]
ipeco85 please support the community by providing the fix to the issue. Problem solved is not a sufficient answer. If you are joining the Linux community, you will find that unlike Micro$oft, this is a giving community, and your feedback on how you solved the problem assists future visitors with the same issue. Please can you post the Fix you achieved here. Thank you.
